Makefile

```diff
@@ -56,6 +56,11 @@ unit-test:
 integration-test:
 	coverage run -m pytest $(COSMPY_TESTS_DIR) --doctest-modules -m "integration"
 
+# Run all third party tests
+.PHONY: third-party-test
+third-party-test:
+	coverage run -m pytest $(COSMPY_TESTS_DIR) --doctest-modules -m "third party"
```

Comment:
This comment seems to failing during test run - the third [see this line](https://github.com/fetchai/cosmpy/actions/runs/5069951654/jobs/9104289269?pr=357#step:5:62) in test run.

Reply:
it kind of feels like that `"third_party"` should be there instead of `"third party"`.

Reply:
It is the same as "not integration" few lines above
```
markers = [
    "integration: marks tests as integration (deselect with '-m \"not integration\"')",
```